comp/core/telemetry/telemetryimpl/telemetry.go

```diff
@@ -85,8 +108,7 @@ func (t *telemetryImpl) Handler() http.Handler {
 func (t *telemetryImpl) Reset() {
 	mutex.Lock()
 	defer mutex.Unlock()
-	registry = prometheus.NewRegistry()
-	t.registry = registry
+	t.registry = newRegistry()
```

Comment:
Now the global `registry` is not reset, is that on purpose ?

Reply:
Ohh right 🤦

Reply:
Forgot about the global var `registry`